angular-monorepo/README.md

```diff
@@ -0,0 +1,63 @@
+# AngularMonorepo
```

Comment:
Can we make sure these match the README layout we have for all these recipes? https://github.com/nrwl/nx-recipes/blob/main/CONTRIBUTING.md#example-structure

Here's the example for the Angular Standalone: https://github.com/nrwl/nx-recipes/tree/main/angular-standalone (we probably need to remove the STackblitz buttons here as well again for now, unfortunately)


Same for the React Monorepo: https://github.com/nrwl/nx-recipes/tree/main/react-monorepo